README.md

```diff
@@ -161,18 +161,17 @@ In addition, you may:
 
 ## License
 
-GreptimeDB uses the [Apache 2.0 license][1] to strike a balance between
+GreptimeDB uses the [Apache License 2.0](https://apache.org/licenses/LICENSE-2.0.txt) to strike a balance between
 open contributions and allowing you to use the software however you want.
 
-[1]: <https://github.com/greptimeTeam/greptimedb/blob/main/LICENSE>
-
 ## Contributing
 
 Please refer to [contribution guidelines](CONTRIBUTING.md) for more information.
 
 ## Acknowledgement
-- GreptimeDB uses [Apache Arrow](https://arrow.apache.org/) as the memory model and [Apache Parquet](https://parquet.apache.org/) as the persistent file format.
+
+- GreptimeDB uses [Apache Arrow™](https://arrow.apache.org/) as the memory model and [Apache Parquet™](https://parquet.apache.org/) as the persistent file format.
 - GreptimeDB's query engine is powered by [Apache Arrow DataFusion](https://github.com/apache/arrow-datafusion).
```

Comment:
What about this? "Apache Arrow™ DataFusion" or "Apache Arrow™ DataFusion™" or leave it as is

Reply:
Somehow DataFusion is seeking to be a TLP but I agree that we can use Apache Arrow DataFusion™ now as https://arrow.apache.org/datafusion/ writes "Apache Arrow DataFusion, Arrow DataFusion, ... are either registered trademarks or trademarks of The Apache Software Foundation in the United States and other countries."

Reply:
```suggestion
- GreptimeDB's query engine is powered by [Apache Arrow DataFusion™](https://github.com/apache/arrow-datafusion).
```